### DISCUSSION TITLE
argocd-redis containner dont start

### DISCUSSION THREAD

**Discussion by andygit88** [argoproj/argo-cd#6738] [Q&A]
I have a Vagrant with SUSE kubectl. Argocd-redis  does not work. Other servers are working fine, like the deployment syncs with github, and respond to changed on github.  Because of the argocd-redis server is not working, the flowchart does not show any svc and po; even thought the deployment is running.

**Comment by jgwest** [comment]
Hi @andywei88, you may want to provide more details here that may be useful for diagnosis.
